### PR TITLE
Disable crossdomainrpc_test

### PR DESCRIPTION
**Pull request by joeltine**
Flaky on IE

### CODE REVIEW

File: protractor_spec.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ var IGNORED_TESTS = [
   'closure/goog/net/jsloader_test.html',
   'closure/goog/net/filedownloader_test.html',
   'closure/goog/promise/promise_test.html',
-  'closure/goog/editor/plugins/abstractdialogplugin_test.html'
+  'closure/goog/editor/plugins/abstractdialogplugin_test.html',
```
Comment (concavelenz) [inline protractor_spec.js]: I'm a little unclear on why things are continually being added to this list.

Comment (joeltine) [inline protractor_spec.js]: They're the tests that have promise timeouts that fail before the set timeout value. I just add them to this list as they come up.

Comment (joeltine) [inline protractor_spec.js]: Obviously, this is meant to be temporary.

Comment (Dominator008) [inline protractor_spec.js]: I wonder if `closure/goog/editor/plugins/abstractdialogplugin_test.html` is fixed by the following commit:
https://github.com/google/closure-library/commit/a1b20191b1cfd7937b1a3402eb1347bd54bc3116#diff-c0a7f4c78a87dc0d9c753419bf2fd040R89

**Pull request merged**